packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/renderer/edge/EdgeLayout.types.ts

```diff
@@ -25,7 +25,9 @@ export type GetUpdatedConnectionHandlesParameters = (
   sourcePosition: Position,
   targetPosition: Position,
   sourceHandle: string,
-  targetHandle: string
+  targetHandle: string,
+  nodes: Node<NodeData>[],
+  edges: Edge<EdgeData>[]
```

Comment:
This seems wrong. It's added here so that we can pass it to `getUpdatedConnectionHandles()`, but that function does not actually use these info.

Reply:
Yes I removes this parameters